### ISSUE TITLE
Include babel-polifyll

### ISSUE THREAD

**Issue by apsavin** [researchgate/gemini-babel#4]
...because of async / await support

Any chance that such feature will be added to the plugin?
Looks extremely easy to implement, but would you like to have such a feature?

**Comment by SevInf** [comment]
I agree it should be possible to include it. Will you send a pull request?
